packages/ui/package.json

```diff
@@ -20,13 +20,16 @@
   },
   "dependencies": {
     "@heroicons/react": "2.0.18",
+    "@radix-ui/react-dropdown-menu": "2.0.5",
+    "@radix-ui/react-form": "0.0.3",
+    "@radix-ui/react-tooltip": "1.0.6",
```

Comment:
If we are using Tailwindcss, do we need Radix?

Reply:
Yes, Radix-UI is a headless UI component library (the components have no styling) that has a focus on ease of use and accessibility. Therefore, all I need to do with Radix-UI is the styling the individual components rather than needing to add the required logic and accessibility for these components. [Here](https://www.radix-ui.com/) is their website.